sdk/src/client/node_api/core/mod.rs

```diff
@@ -5,16 +5,31 @@
 
 pub mod routes;
 
+use packable::PackableExt;
+
 #[cfg(not(target_family = "wasm"))]
 use crate::client::constants::MAX_PARALLEL_API_REQUESTS;
 use crate::{
     client::{Client, Result},
-    types::block::output::{OutputId, OutputMetadata, OutputWithMetadata},
+    types::block::output::{Output, OutputId, OutputMetadata, OutputWithMetadata},
 };
 
 impl Client {
+    // Finds output and its metadata by output ID.
+    /// GET /api/core/v3/outputs/{outputId}
+    /// + GET /api/core/v3/outputs/{outputId}/metadata
+    pub async fn get_output_with_metadata(&self, output_id: &OutputId) -> Result<OutputWithMetadata> {
+        let output = Output::unpack_verified(
+            self.get_output_raw(output_id).await?,
+            &self.get_protocol_parameters().await?,
+        )?;
```

Comment:
should this be its own method `get_output`? Seems like something that could be used more than once.

Reply:
I don't think it would be used more than once. But I've added an item to https://github.com/iotaledger/iota-sdk/issues/647 because it joins the discussion about the raw routes and I'm not yet sure what to do about them.